#ifndef DEMO_H
#define DEMO_H

#include <stdlib.h>
#include <stdio.h>
#include <limits.h>
#include <string.h>
#include <assert.h>
#include <time.h>
#include <stdint.h>
#include <iostream>
#include <string>
#include <fstream>
#include <cstdlib>
#include <cctype>

#include <openssl/pem.h>
#include <openssl/evp.h>
#include <openssl/bio.h>
#include <openssl/err.h>
#include <openssl/ec.h>
#include <openssl/rand.h>
#include <openssl/crypto.h>
#include <openssl/sha.h>


#endif